Makefile

```diff
@@ -16,11 +16,14 @@ aspects:
 	# bundle
 	./export_tulsi_aspect_dir.sh ${PWD}/$(ASPECTDIR)
 
+
+# Make a SPM generated Xcode project.
+# cause problems.
 # Copy the tulsi-aspects and XCHammerAssets adjacent to the Xcode build
-# directory to allow loading of resources, since we can't express this in SPM or
-# Xcode generated SPM
-# This is the format of the XCHammer package
-workspace: aspects
+# directory to allow loading of resources, since we can't express this in SPM
+#
+# Note: this is brittle and might not work as expected
+workspace_legacy: aspects
```

Comment:
can we call this `spm_proj`?

Reply:
I like the convention of using `workspace` if that is what they are both calle - we could call it `workspace_spm`